app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -13,3 +13,12 @@ attributes :sockets, :cores, :ram
 attributes :instance_multiplier, :stacking_id, :multi_entitlement
 attributes :type
 attributes :name => :product_name
+attributes :unmapped_guest
+attributes :virt_only
+
+node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
+  {
+    id: subscription.host.uuid,
+    name: subscription.host.name
+  }
+end
```

Comment:
https://github.com/Katello/katello/blob/068062449b30e985695aa6071837290e4afbd3df/engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/views/subscription-type.html#L5 this uses it to show subscription type and guest host subscriptions in the index